securedrop/sass/_base.sass

```diff
@@ -348,9 +366,7 @@
 
   .logo
     width: 250px
-    margin-right: 20px
-    margin-top: 36px
-    margin-left: 9px
+    margin: 20px 36px 20px auto
```

Comment:
Hmm, this produces a weird offset in both Arabic and English:

![screen shot 2017-10-03 at 1 46 50 pm](https://user-images.githubusercontent.com/7832803/31148353-5454b92a-a841-11e7-98b3-f5a224781abd.png)

Suggestion: 

```
    margin-right: 9px
    margin-top: 36px
    margin-left: 9px
```

This renders well in both English:

![screen shot 2017-10-03 at 1 50 45 pm](https://user-images.githubusercontent.com/7832803/31148544-e0e44b94-a841-11e7-8fab-d52ad540b369.png)

And Arabic:

![screen shot 2017-10-03 at 1 50 36 pm](https://user-images.githubusercontent.com/7832803/31148560-e6f60c0c-a841-11e7-9c64-d5106630156d.png)

Other solutions welcome